src/components/nodes/boundaryErrorEvent/boundaryErrorEvent.vue

```diff
@@ -5,9 +5,25 @@
 <script>
 import BoundaryEvent from '@/components/nodes/boundaryEvent/boundaryEvent';
 import errorBoltIcon from '@/assets/boundary-error-event-icon.svg';
+import validBoundaryEventTargets from '@/components/nodes/boundaryEvent/validBoundaryEventTargets';
+import { getAttachedErrorBoundaryEvents } from '@/targetValidationUtils';
 
 export default {
   extends: BoundaryEvent,
+  methods: {
+    doesNotHaveOtherBoundaryEvents(model) {
+      const boundaryEvents = getAttachedErrorBoundaryEvents(model).filter(boundaryEvent => {
+        return boundaryEvent !== this.shape;
+      });
+
+      return boundaryEvents.length === 0;
```

Comment:
This could be reduced to the following:
```suggestion
      return getAttachedBoundaryEvents(model)
          .filter(boundaryEvent => boundaryEvent !== this.shape)
          .length === 0;
```
No need for the intermediate variable.